3rdparty/python/requirements.txt

```diff
@@ -8,11 +8,6 @@ ansicolors==1.1.8
 chevron==0.14.0
 fasteners==0.16.3
 freezegun==1.2.1
-
-# Note: we use humbug to report telemetry. When upgrading, ensure the new version maintains the
-# anonymity promise we make here: https://www.pantsbuild.org/docs/anonymous-telemetry
```

Comment:
Could make a doc update to remove that page as well.

Reply:
Well, not if we keep the machinery around, as you suggest.

Reply:
Also, hide the page. Don't delete. For people using old versions of Pants where we did still have the warning